<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>32</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>344.000000000000000</width>
      <height>344.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QPushButton" name="indexA">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>9</y>
      <width>49</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="indexB">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>9</y>
      <width>49</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="indexC">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>9</y>
      <width>49</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="indexD">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>9</y>
      <width>49</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="indexE">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>9</y>
      <width>49</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="indexF">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>9</y>
      <width>49</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="indexG">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>326</x>
      <y>9</y>
      <width>49</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="index7">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>34</y>
      <width>23</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="index6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>83</y>
      <width>23</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="index5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>132</y>
      <width>23</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="index4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>181</y>
      <width>23</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="index3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>230</y>
      <width>23</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="index2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>279</y>
      <width>23</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="index1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>328</y>
      <width>23</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>201</width>
      <height>363</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="turnLabel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Tura</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="undoButton">
         <property name="text">
          <string>Cofnij</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="endTurnButton">
         <property name="text">
          <string>Zakończ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="historyLabel">
       <property name="text">
        <string>Historia Gry</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="prevTurnButton">
         <property name="text">
          <string>Do tyłu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextTurnButton">
         <property name="text">
          <string>Do przodu</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="cheatsLabel">
       <property name="text">
        <string>Oszustwa</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="turnEditButton">
         <property name="text">
          <string>Edytuj Planszę</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hintButton">
         <property name="text">
          <string>Podpowiedź</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>Bieżąca Tura</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="statusDisplay">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Gra</string>
    </property>
    <widget class="QMenu" name="menuNewGame">
     <property name="title">
      <string>Nowa Gra</string>
     </property>
     <addaction name="actionHuman_vs_Human"/>
     <addaction name="actionAI_vs_Human"/>
     <addaction name="actionHuman_vs_AI"/>
     <addaction name="actionAI_vs_AI"/>
    </widget>
    <addaction name="menuNewGame"/>
    <addaction name="separator"/>
    <addaction name="actionSaveGame"/>
    <addaction name="actionLoadGame"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionGameRules"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHuman_vs_Human">
   <property name="text">
    <string>Człowiek vs Człowiek</string>
   </property>
  </action>
  <action name="actionAI_vs_Human">
   <property name="text">
    <string>Komputer vs Człowiek</string>
   </property>
  </action>
  <action name="actionHuman_vs_AI">
   <property name="text">
    <string>Człowiek vs Komputer</string>
   </property>
  </action>
  <action name="actionAI_vs_AI">
   <property name="text">
    <string>Komputer vs Komputer</string>
   </property>
  </action>
  <action name="actionSaveGame">
   <property name="text">
    <string>Zapisz Grę</string>
   </property>
  </action>
  <action name="actionLoadGame">
   <property name="text">
    <string>Wczytaj Grę</string>
   </property>
  </action>
  <action name="actionGameRules">
   <property name="text">
    <string>Zasady Gry</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
